renovate.json

```
@@ -7,11 +7,11 @@
   "automergeStrategy": "auto",
   "automergeType": "pr",
   "prConcurrentLimit": 5,
-  "commitMessagePrefix": "NO-JIRA: ",
   "ignoreTests": false,
   "rebaseLabel": "needs-rebase",
   "rebaseWhen": "behind-base-branch",
   "recreateWhen": "always",
+  "commitMessageSuffix": "{{baseBranch}}",
```

```suggestion
  "commitMessagePrefix": "{{baseBranch}}",
```

I think you want the prefix and not suffix. ie (release-0.11 - ...)

---

https://docs.renovatebot.com/configuration-templates/ says specifically to not touch the prefix as other things use it.  It also says that suffix is used for things like branches.